pkgs/development/python-modules/uni-curses/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, ncurses, x256, setuptools }:
+
+buildPythonPackage rec {
+  pname = "unicurses";
```

Comment:
pname is `unicurses` but attribute path is `uni-curses`, these should match

Reply:
Whoops, corrected now.